### PR TITLE
Add Dockerhub Credentials to CI

### PR DESCRIPTION
**Pull request by gsheni**
- CI before authentication
Screen Shot 2020-10-28 at 12 28 11 PM

- CI after authentication:
Screen Shot 2020-10-28 at 12 31 33 PM

### CODE REVIEW

**Review by rwedge — APPROVED** [review]
Looks good!

**Comment by gsheni** [comment]
After using CircleCI context for credentials
Screen Shot 2020-10-28 at 1 54 07 PM

File: .circleci/config.yml [hunk 1]
```diff
@@ -311,16 +333,22 @@ workflows:
...
+            - dockerhub
       - run_premium_primitives_tests_on_release:
           name: "run premium primitives tests on release"
           filters:
             branches:
               ignore: /.*/
             tags:
               only: /^v\d+\.\d+\.\d+$/
+          context:
+            - dockerhub
```
Comment (rwedge) [inline .circleci/config.yml:354]: i think we can write this as `context: dockerhub`

Comment (gsheni) [reply .circleci/config.yml:354]: Fixed

**Review by rwedge — APPROVED** [review]
:shipit:

**Pull request merged**